### ISSUE TITLE
why sequences in provided train_parent.txt and DAVIS2016/train.txt are not the same?

### ISSUE THREAD

**Issue by nijkah** [scaelles/OSVOS-TensorFlow#15]
I guess parent model file provided from README is trained on your own sequences written in train_parent.txt.
Is there any purpose the provided model is trained on your own sequences?

Or is just train_parent.txt example of training sequences?

**Comment by nijkah** [comment]
I think the provided model is trained on DAVIS/train_seqs.txt
And train_parent.txt in this repository is just example training sequences.

**Comment by kmaninis** [comment]
Hi, this is the training set of DAVIS-2016. In the results reported in the paper, we used the training set to train the parent model, and the validation set to finetune on the first mask.

**Comment by scaelles** [comment]
If you are referring to the `train.txt` in `DAVIS/ImageSets/480p`, the only difference with the `train_parent.txt` is that in the latter the `/` preceding the paths is removed. The sequences are the same in both files and they are the ones in the train set of DAVIS 2016. Those sequences are the ones used to train the parent model provided in the README. In this way, the model learns a notion of object saliency that improves the results in the final fine tunning. Let us know if you have any further question.
